src/ConnectedMachine/ConnectedMachine.Autorest/test/Update-AzConnectedMachineRunCommand.Tests.ps1

```diff
@@ -0,0 +1,35 @@
+if(($null -eq $TestName) -or ($TestName -contains 'Update-AzConnectedMachineRunCommand'))
+{
+  $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
+  if (-Not (Test-Path -Path $loadEnvPath)) {
+      $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
+  }
+  . ($loadEnvPath)
+  $TestRecordingFile = Join-Path $PSScriptRoot 'Update-AzConnectedMachineRunCommand.Recording.json'
+  $currentPath = $PSScriptRoot
+  while(-not $mockingPath) {
+      $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
+      $currentPath = Split-Path -Path $currentPath -Parent
+  }
+  . ($mockingPath | Select-Object -First 1).FullName
+}
+
+Describe 'Update-AzConnectedMachineRunCommand' {
+    It 'UpdateExpanded' -skip {
+        { throw [System.NotImplementedException] } | Should -Not -Throw
```

Comment:
Please remove the place holders for the parameter sets that are not necessary.

Reply:
fixed